src/main/java/uz/hiparts/hipartsuz/controller/TelegramUserController.java

```diff
@@ -0,0 +1,51 @@
+package uz.hiparts.hipartsuz.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uz.hiparts.hipartsuz.model.TelegramUser;
+import uz.hiparts.hipartsuz.model.enums.UserState;
+import uz.hiparts.hipartsuz.service.TelegramUserService;
+
+@RequiredArgsConstructor
+@RestController("/api/v1/tg-user")
+public class TelegramUserController {
+    private final TelegramUserService telegramUserService;
+
```

Comment:
Telegram user doesn't need a controller because telegram user is only for telegram bot logic